module-1/assignments/activity1.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="utf-8"?>​
+<messages>
+ <message>
```

Comment:
these tag names don't make sense here
could have used people > person for example

Reply:
Alright sir noted.. if i'm right, where the information about myself is i could have used people or person?